llm_studio/src/utils/modeling_utils.py

```diff
@@ -1070,3 +1073,12 @@ def generate(backbone, batch, cfg, streamer, remove_prompt=True):
     if remove_prompt:
         output = output[:, input_ids.shape[1] :]
     return output
+
+
+def get_torch_dtype(dtype):
+    if dtype == "float16":
+        return torch.float16
+    elif dtype == "bfloat16":
+        return torch.bfloat16
+    else:
+        return torch.float32
```

Comment:
Not valid input for autocast. Should raise an error instead.

Reply:
indeed, but I was thinking of reusing that function for something else potentially to go frokm string to torch dtype